kiwi/system/uri.py

```diff
@@ -66,18 +71,16 @@ def __init__(self, uri, repo_type=None):
             'http': True,
             'https': True,
             'ftp': True,
-            'obs': True,
-            'ibs': True
+            'obs': True
         }
         self.local_uri_type = {
             'iso': True,
             'dir': True,
             'file': True,
-            'suse': True,
             'obsrepositories': True
         }
 
-    def translate(self):
+    def translate(self, check_build_environment=True):
         """
         Translate repository location according to their URI type
 
```

Comment:
nit-picking, but I would update the docs to include check_build_environment parameter explanation